test/unit/lib/data_hygiene/publishing_api_document_republisher_test.rb

```diff
@@ -30,17 +30,4 @@ class DataHygiene::PublishingApiDocumentRepublisherTest < ActiveSupport::TestCas
       DataHygiene::PublishingApiDocumentRepublisher.new(Organisation)
     end
   end
-
-  test "unpublishes if the document is unpublished" do
-    document = create(:document, content_id: SecureRandom.uuid)
-    edition = create(:unpublished_edition, title: "Unpublished edition", document:)
-    unpublishing = edition.unpublishing
-
-    PublishingApiUnpublishingWorker.expects(:new).returns(worker = mock)
-    worker.expects(:perform).with(unpublishing.id, true)
-
-    Sidekiq::Testing.inline! do
-      DataHygiene::PublishingApiDocumentRepublisher.new(edition.class, NullLogger.instance).perform
-    end
-  end
```

Comment:
This is essentially a duplicate of the test in `test/unit/app/workers/publishing_api_document_republishing_worker_test.rb`